server/src/test/java/org/elasticsearch/indices/state/CloseIndexIT.java

```diff
@@ -56,6 +59,14 @@
 
     private static final int MAX_DOCS = 25_000;
 
+    @Override
+    public Settings indexSettings() {
+        Settings.builder().put(super.indexSettings())
+            .put(IndexSettings.INDEX_TRANSLOG_FLUSH_THRESHOLD_SIZE_SETTING.getKey(),
+                new ByteSizeValue(randomIntBetween(1, 100 * 1024), ByteSizeUnit.KB));
```

Comment:
Did you intend to have such a large range? Up to 100 MB means most of the time there’s no flush during these tests.

Reply:
Good catch! 4MB is more realistic since we are indexing a small number of documents in this suite.